pkg/state/license.go

```diff
@@ -0,0 +1,144 @@
+package state
+
+import (
+	"errors"
+	"fmt"
+
+	memdb "github.com/hashicorp/go-memdb"
+	"github.com/kong/go-database-reconciler/pkg/utils"
+)
+
+const (
+	licenseTableName = "license"
+)
+
+var licenseTableSchema = &memdb.TableSchema{
```

Comment:
I wonder what are others' thoughts on this but I'd personally change this to a `func` returning this variable. This way we don't have issues with sharing the same underlying memory area.